### PR TITLE
[NimManager] improvize tunerTypeChanged write more info in log

### PR DESCRIPTION
**Pull request by Dima73**

### CODE REVIEW

File: lib/python/Components/NimManager.py [hunk 1]
```diff
@@ -1545,41 +1545,53 @@ def createATSCConfig(nim, x):
...
 	def tunerTypeChanged(nimmgr, configElement, initial=False):
-		configElement.save()
 		fe_id = configElement.fe_id
 		eDVBResourceManager.getInstance().setFrontendType(nimmgr.nim_slots[fe_id].frontend_id, nimmgr.nim_slots[fe_id].getType())
 		try:
-			frontend = eDVBResourceManager.getInstance().allocateRawChannel(fe_id).getFrontend()
-			if not os.path.exists("/proc/stb/frontend/%d/mode" % fe_id) and frontend.setDeliverySystem(nimmgr.nim_slots[fe_id].getType()):
+			raw_channel = eDVBResourceManager.getInstance().allocateRawChannel(fe_id)
+			if raw_channel is None:
+				self.session.nav.stopService()
```
Comment (athoik) [inline lib/python/Components/NimManager.py]: Are you missing a `raw_channel = eDVBResourceManager.getInstance().allocateRawChannel(fe_id)` after stopService?

**Comment by Dima73** [comment]
Maybe there is a record or stream or PiP.

**Comment by athoik** [comment]
```patch
+			raw_channel = eDVBResourceManager.getInstance().allocateRawChannel(fe_id)
+			if raw_channel is None:
+				self.session.nav.stopService()
+				if raw_channel is None:
+					print "[InitNimManager] %d: tunerTypeChanged to '%s' failed (BUSY)" %(fe_id, configElement.getText())
+					return
```

raw_channel is None (true) -> stopService -> raw_channel is None (true) -> fail BUSY...

Did you mean the following?

```patch
+			raw_channel = eDVBResourceManager.getInstance().allocateRawChannel(fe_id)
+			if raw_channel is None:
+				self.session.nav.stopService()
+				raw_channel = eDVBResourceManager.getInstance().allocateRawChannel(fe_id)
+				if raw_channel is None:
+					print "[InitNimManager] %d: tunerTypeChanged to '%s' failed (BUSY)" %(fe_id, configElement.getText())
+					return
```

**Comment by Dima73** [comment]
Yep.Thanks.

**Pull request merged**